src/components/icon/GraphItemIcon/GraphItemIcon.tsx

```diff
@@ -86,7 +86,7 @@ export const GraphItemIcon: React.FC<{ type: string }> = ({ type }) => {
   }
 
   return <div className={css?.graphItemtIconTypeDiv}>
-    <img alt="Graph Item Icon" src={icon} width="22" height="22" />
+    <AppIcon app="@folio/workflow" iconKey="app" size="small" src={icon} />
```

Comment:
I went ahead and created a Tech Debt issue to address the introduction of the use of `AppIcon` here.
- https://folio-org.atlassian.net/browse/UIWRKFLOW-38

Reply:
Not sure why there are two icon components. Is is that AppIcon is only expected to work in the context of the header navbar and Icon should be used elsewhere? The AppIcon does work as expected in this context but is preferred to follow which the framework prescribes.

Thanks for the follow up issue.

Reply:
Here is another reference use of `AppIcon`. Again, still not sure what this component is for or how it differs from the `Icon` component.

https://github.com/folio-org/stripes-components/blob/b0ec27770e4a32c50d4e0af8e871391158535035/lib/PaneHeader/readme.md

Perhaps we should create a WorkflowNodeIcon similar to CustomIcon here https://github.com/folio-org/stripes-components/blob/b0ec27770e4a32c50d4e0af8e871391158535035/lib/Icon/stories/CustomIcon.js#L4?

Reply:
It has been decided `AppIcon` is the correct component to use here.